src/components/accessories/admin/users/usersTable/UsersTable.tsx

```diff
@@ -49,6 +50,7 @@ export const UsersTable = ({ headerActions }: IOwnProps) => {
         userGroupName:
           item.userGroupName?.desc ?? item.userGroupName?.code ?? "",
         desc: item.desc ?? "",
+        passwd: item.passwd ?? ""
```

Comment:
Is the user password really sent (in plain text) to front-end (in the DTO) 😨 ?

Reply:
Yes, and it is stored in plain text in the database, which is not a best practice.
As the app is supposed to be used offline, I guess that's ok for now 😬

Reply:
Sorry, where is saved in plain text in the database? it should not... it should be encrypted and only encrypted password should be sent to the BE for comparing

Reply:
Or at least in this way is done using GUI that rely on the same tables

Reply:
Up!

Reply:
Exactly @mwithi . As @SteveGT96 is working on OH2-253, I was drawing his attention to this point. Password, even encrypted, should not be sent to FE

Reply:
I get a "not allowed" error when changing a user's group, so I think we have to wait for OH2-253 to be merged

![image](https://github.com/user-attachments/assets/a83f57d5-9ea0-4138-a3fa-8271159d1169)